st7735r.h

```diff
@@ -30,6 +31,16 @@
 
 #define DISPLAY_NATIVE_COVERED_LEFT_SIDE 2
 #define DISPLAY_NATIVE_COVERED_TOP_SIDE 1
+#elif defined(ADAFRUIT_ST7789_PITFT_114)
+// The Adafruit PiTFT 1.14" uses ST7789, but it has some quirks, such as not working with the chip select signal toggle,
+// and having a unique offset in display memory.
+#include "pitft_114r_st7789.h"
+#define DISPLAY_NATIVE_WIDTH 240
+#define DISPLAY_NATIVE_HEIGHT 320
+#define DISPLAY_NATIVE_COVERED_TOP_SIDE 40
+#define DISPLAY_NATIVE_COVERED_BOTTOM_SIDE 40
+#define DISPLAY_NATIVE_COVERED_LEFT_SIDE 53
+#define DISPLAY_NATIVE_COVERED_RIGHT_SIDE 52
```

Comment:
This reads like you chose in /boot/config.txt to have `hdmi_cvt=320 240 60 1 0 0 0` for a 320x240 display mode, and then here are opting to crop a centered subrectangle of 240x135 pixels in size to display? This means that the full display output buffer will not be displayed, but only a centered subrectangle will be. What if a user wanted to display the full display area?

Would it be better to have here by default
```
#define DISPLAY_NATIVE_WIDTH 135
#define DISPLAY_NATIVE_HEIGHT 240
```
and no cropping. And then in /boot/config.txt have
```
hdmi_cvt=240 135 60 1 0 0 0
```
to get the full screen display outputted? For e.g. console use cases, that would probably be a sensible default?